### ISSUE TITLE
[Feature Request] Turn off display from command line

### ISSUE THREAD

**Issue by Recontaminated** [xanderfrangos/twinkle-tray#398]
Option to shutdown monitor in .exe flags.
For example
`"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" --MonitorNum=1 --turnOff` will turn off the displays according to the method set in Turn off Displays action.

Also does Twinkle try attempt to turn the displays back on using DCC/CI when set to hardware signal?

**Comment by KiwiMonster364** [comment]
+1

**Comment by xanderfrangos** [comment]
This feature is available as of v1.14.4. 👍

Instead of the suggested `--turnOff` flag, you'll need to set the VCP code for turning off the display (`0xD6`). So the full command would look like:
```
"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" --MonitorNum=1 --VCP="0xD6:5"
```
or
```
"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" --MonitorNum=1 --VCP="0xD6:4"
```
...depending on if your display works better with a value of 5 or 4.

If your display supports turning it back on, the command would be:
```
"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" --MonitorNum=1 --VCP="0xD6:1"
```
